sheet_to_triples/field.py

```diff
@@ -92,6 +92,10 @@ def as_slug(self):
     def as_uc(self):
         return _must(self._pattern.sub('', _str(self._value).title()))
 
+    @property
+    def as_lc(self):
+        return self.as_uc[0].lower() + self.as_uc[1:]
```

Comment:
Noo, though `.as_uc` is pure, Python doesn't know anything about that, so using the attribute twice is calling the method twice. So put in a local var first.